db/migrate/20161128184121_add_creator_id_and_renter_id_to_rentals.rb

```diff
@@ -0,0 +1,7 @@
+class AddCreatorIdAndRenterIdToRentals < ActiveRecord::Migration[5.0]
+  def change
+    add_column :rentals, :creator_id, :integer
+    add_column :rentals, :renter_id, :integer
+    remove_column :rentals, :user_id # no longer needed
```

Comment:
Specifying a column type is optional here, but I think it's preferable in general, so that this migration could be rolled back if needed.